book/dune

```diff
@@ -1,12 +1,305 @@
+(alias
```

Comment:
This looks like a rebase/merge mishap, `book/dune` shouldn't be impacted by your changes and in particular here it seems to revert some of the work we've done on the HTML and LaTeX rules.